lms/templates/problem.html

```diff
@@ -36,7 +36,10 @@ <h3 class="hd hd-2 problem-header" tabindex="-1">
     % endif
     % if answer_available:
     <span class="problem-action-button-wrapper">
-        <button class="show problem-action-btn btn-default btn-small"><span class="icon fa fa-exclamation-triangle" aria-hidden="true"></span><span class="show-label">${_('Show Answer')}</span></button>
+        <button class="show problem-action-btn btn-default btn-small">
+            <span class="icon fa fa-exclamation-triangle" aria-hidden="true"></span><span class="show-label">${_('Show Answer')}</span>
+            <span class="show-answer-tooltip sr">${_('To get credit, only select after Submit.')}</span>
```

Comment:
@catong to @chris-mike 's question in this PR, I agree in some circumstances the tooltip does not provide a clear message. For example, when Show Answer is configured to show only after 1 attempt, you already clicked Submit, Show Answer button shows up, that tooltip is not meaningful (as you already clicked Submit).

What about "To get credit, only select after Submit if no attempts left." Better (or worse) yet, "You will not receive credit once you see the answer."